mmselfsup/models/backbones/resnet.py

```diff
@@ -86,7 +86,7 @@ def __init__(self,
                  norm_cfg=dict(type='BN', requires_grad=True),
                  norm_eval=False,
                  with_cp=False,
-                 zero_init_residual=True,
+                 zero_init_residual=False,
```

Comment:
why do we change this?

Reply:
Because methods like rotnet, relativeloc, moco, etc., do not use zero_init_residual. Setting this to True will affect these methods' behaviors. In our previous release, zero_init_residual is set to False as default.